services/frontend/src/store/slices/profiles.slice.js

```diff
@@ -7,11 +7,17 @@ export const profilesSlice = createSlice({
     initialState: {},
     reducers: {
         setProfiles: (state, action) => action.payload,
+        setProfile: (state, action) => {
```

Comment:
... there is quite literally an equivalent `addProfile` method right below this

Reply:
Yep. I created `setProfile` since it implies a different type of operation, therefore making the code easier to read.

That said, addProfile isn't being used for anything right now so I just deleted it and that can be a problem for later.